doc/classes/SceneTree.xml

```diff
@@ -55,6 +45,12 @@
 			<description>
 				Changes the running scene to a new instance of the given [PackedScene].
 				Returns [constant OK] on success or [constant ERR_CANT_CREATE] if the scene cannot be instantiated.
+				[codeblock]
+				get_tree().change_scene_to(load("res://my_scene.tscn"))
+				# Defers the loading process to occur as soon as the script is compiled.
```

Comment:
I don’t think that “defer” is the right word to use here. When you defer something, you cause that thing to happen later (this is how `call_deferred()` and `set_deferred()` use the word). Using `preload` causes the loading to happen _earlier_, not later. To make the example more clear, I would change it to:

```gdscript
# Loads the scene when the game reaches this line of code.
get_tree().change_scene_to(load("res://my_scene.tscn"))

# Loads the scene when this script is compiled. May not always work due to
# cyclic reference issues. If in doubt, use `load()`.
get_tree().change_scene_to(preload("res://my_scene.tscn"))
```